sdk/search/Azure.Search.Documents/src/Options/SearchOptions.cs

```diff
@@ -130,7 +130,7 @@ internal string SelectRaw
         internal string OrderByRaw
         {
             get => OrderBy.CommaJoin();
-            set => OrderBy = SearchExtensions.CommaSplit(value);
+            set => OrderBy = SearchExtensions.CommaSplit(value, hasODataFunctions: true);
```

Comment:
In Track 1, we avoided this class of problem by keeping duplicate models for search parameters -- one Swagger-generated that treated these as plain strings, and another input-only hand-crafted model that was only used on input. That way, we only ever put the commas in -- we never had to take them out. :)

Reply:
I like that approach - at least the concept of not having to parse out commas. I haven't gotten to the implementation change called here just yet, but parsing out non-delimiting characters (e.g. quotes out of "fields" in a CSV) is often non-trivial.